.github/workflows/docker-build-static.yml

```diff
@@ -425,6 +425,16 @@ jobs:
     needs: [build-ui, build-bin]
     name: Build Docker image
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container: ["debian","alpine"]
+        include:
+          - container: alpine
+            platforms: linux/amd64,linux/arm64
+            suffix: ""
```

Comment:
```suggestion
            suffix: "-alpine"
```

Reply:
If `-alpine` enabled , need metadata tag to only produce `v{{version}}` only.

Reply:
see the main comment i left with this batch - we'd need some extra stuff to get both plain and -alpine tags